src/platform/packages/shared/kbn-esql-ast/src/commands_registry/types.ts

```diff
@@ -140,6 +140,7 @@ export interface ICommandContext {
   variables?: ESQLControlVariable[];
   supportsControls?: boolean;
   histogramBarTarget?: number;
+  hasMinimumLicenseRequired?: (license?: ESQLLicenseType) => boolean;
```

Comment:
I don't like the fact that we have it both in context and callbacks. This is a callback and it makes more sense to be there.

Reply:
Yes you have to pass it to the validation and we havent decided yet how the final api will look like so at this point I prefer to have the 2 apis consistent